adobe/react-spectrum
# Fix RAC Button prop table description rendering

Hello, I've noticed the description of `form` prop of [Button](https://react-spectrum.adobe.com/react-aria/Button.html#props) doesn't render properly. 

The problem here is, the substring `<form>` in the description is actually parsed and rendered as a real HTML form tag by `<Markdown />` component in `packages/dev/docs/src/types.js`.

https://github.com/adobe/react-spectrum/blob/86b38c87868ce7f262e0df905e5ac4eb2653791d/packages/react-aria-components/src/Button.tsx#L46-L50

It looks like we can prevent this by specifying the option [`disableParsingRawHTML`](https://www.npmjs.com/package/markdown-to-jsx#optionsdisableparsingrawhtml) to `true`, which is what this PR does.

If using `disableParsingRawHTML` feels too restrictive, we could just turn `<form>` to `` `<form>` `` in the description above.

### Screenshot
<img width="1920" alt="Screenshot 2023-12-21 at 10 40 56 PM" src="https://github.com/adobe/react-spectrum/assets/71210554/5f4827a3-bcae-48d9-8b6d-bf8391fcc3fa">


## ✅ Pull Request Checklist:


## 📝 Test Instructions:

1. Visit http://localhost:1234/react-aria/Button.html#props
2. See if the description of `form` prop renders correctly.

## 🧢 Your Project:
